service-app/internal/api/service.go

```diff
@@ -116,3 +120,16 @@ func (s *Service) CreateCaseStub(ctx context.Context) (*types.ScannedCaseRespons
 
 	return &scannedResponse, nil
 }
+
+func deputyDocType(docType string) string {
+	switch docType {
+	case "FINDOCS":
+		return "Report - Financial evidence"
+	case "DEPREPORTS":
+		return "Report - General"
+	case "DEPCORRES":
+		return "Report"
+	default:
+		return docType
+	}
+}
```

Comment:
This translation is [already done in Sirius](https://github.com/ministryofjustice/opg-sirius/blob/7b0516c82ecab6ded86ccb5f2343d8bc0c5cb404/back-end/module/Application/src/Model/CommandHandler/Scanning/CreateDocumentHandler.php#L51-L58) and I think it makes sense to leave it there so that opg-scanning is dealing with the contract with Restore, and opg-sirius handles how things should be rendered for the business.